migrate-from-tidb-to-mysql.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Migrate Dta from TiDB to MySQL-compatible Databases
+summary: Learn how to migrate data from TiDB to MySQL-compatible databases.
+---
+
+# Migrate Dta from TiDB to MySQL-compatible Databases
+
+This document describes how to migrate data from TiDB clusters to MySQL-compatible databases, such as Aurora, MySQL, and MariaDB. The whole process contains four steps:
+
+1. Set up the environment.
+2. Migrate full data.
+3. Migrate incremental data.
+4. Switch services to the new TiDB cluster.
+
+## Step 1. Set up the environment
+
+1. Deploy a TiDB cluster upstream.
+
+    Deploy a TiDB cluster by using TiUP Playground. For more information, refer to [Deploy and Maintain an Online TiDB Cluster Using TiUP](/tiup/tiup-cluster.md).
+
+    ```shell
+    # Create a TiDB cluster
+    tiup playground --db 1 --pd 1 --kv 1 --tiflash 0 --ticdc 1
+    # View cluster status
+    tiup status
+    ```
+
+2. Deploy a MySQL instance downstream.
+
+    - In a lab environment, you can use Docker to quickly deploy a MySQL instance by running the following command:
+
+        ```shell
+        docker run --name some-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -p 3306:3306 -d mysql
+        ```
+
+    - In a production environment, you can deploy a MySQL instance by following instructions in [Installing MySQL](https://dev.mysql.com/doc/refman/8.0/en/installing.html).
+
+3. Simulate service workload.
+
+    In the lab environment, you can use `go-tpc` to write data to the TiDB cluster upstream. This is to generate event changes in the TiDB cluster. Run the following command to create a database named `tpcc` in the TiDB cluster, and then use TiUP bench to write data to this database.
+
+    ```shell
+    tiup bench tpcc -H 127.0.0.1 -P 4000 -D tpcc --warehouses 4 prepare
+    tiup bench tpcc -H 127.0.0.1 -P 4000 -D tpcc --warehouses 4 run --time 300s
+    ```
+
+    For more details about `go-tpc`, refer to [How to Run TPC-C Test on TiDB](/benchmark/benchmark-tidb-using-tpcc.md).
+
+## Step 2. Migrate full data
+
+After setting up the environment, you can use [Dumpling](/dumpling-overview.md) to export the full data from the upstream TiDB cluster.
+
+1. Disable Garbage Collection (GC).
+
+    To ensure that newly written data is not deleted during incremental migration, you should disable GC for the upstream cluster before exporting full data. In this way, history data is not deleted.
+
+    ```sql
+    MySQL [test]> SET GLOBAL tidb_gc_enable=FALSE;
+    Query OK, 0 rows affected (0.01 sec)
+    MySQL [test]> SELECT @@global.tidb_gc_enable;
+    +-------------------------+：
+    | @@global.tidb_gc_enable |
+    +-------------------------+
+    |                       0 |
+    +-------------------------+
+    1 row in set (0.00 sec)
+    ```
+
+    > **Note:**
+    >
+    > In production clusters, exporting data with GC disabled might affect cluster performance. It is recommended that you export data in off-peak hours.
```

Comment:
I think it's OK to put this note here. Because:
1. The operation related to this note is to disable GC. If we separate the note and the context, it may look weird.
2. This note recommends exporting data during off-peak hours and the next step is exporting data. Users can be well notified.

Reply:
> If this task affects online application, suggest putting it at the beginning of the doc.

请问这样做的理由是什么

Reply:
如果这个任务会影响到线上业务，这类 note 应该放在靠前的位置，让用户在开始前就注意到。避免用户操作到一半，才发现可能当前时间点不适合进行迁移。

Reply:
不建议调整这个note的位置，可以在文章开头新增一个 note